docker/README.md

```diff
@@ -1,120 +1,177 @@
+used to simulate transactions on the Celestia network.
+The `txsim` Docker image is designed to run the `txsim` binary with a
+variety of configurable options.
 
 ## Prerequisites
 
-Before you can use the `txsim` Docker image, you must have a prefunded account set up. The `txsim` binary requires a prefunded account to function correctly. The keyring for this account should be stored in a file that can be accessed by the Docker container.
+- [Docker Desktop for Mac or Windows](https://docs.docker.com/get-docker) or
+[Docker Engine for Linux](https://docs.docker.com/engine/install/)
+and a basic understanding of Docker.
+
+- A prefunded account set up with the keyring stored in a file,
+to be accessed by an instance of the docker image.
 
-## Running the Docker Image
+## Quick-Start
 
-### Docker Run
+1. In your local machine, navigate to the home directory
 
-You can run the `txsim` Docker image using the `docker run` command. Here's an example:
+   ```bash
+   cd $HOME
+   ```
 
-```bash
-docker run -it -v ${HOME}/.celestia-app:/home/celestia ghcr.io/celestiaorg/txsim -k 0 -r http://consensus-validator-robusta-rc6.celestia-robusta.com:26657,http://consensus-full-robusta-rc6.celestia-robusta.com:26657 -g consensus-validator-robusta-rc6.celestia-robusta.com:9090 -t 10s -b 10 -d 100 -e 10
-```
+2. Create a file in which the keyring would be stored.
+The file would be mounted as a volume into the docker container.
 
-In this command, the `-v` option is used to mount the `${HOME}/.celestia-app` directory from the host to the `/home/celestia` directory in the Docker container. This allows the `txsim` binary to access the keyring for the prefunded account.
+   ```bash
+   touch .celestia-app
+   ```
 
-### Docker Compose
+3. Using a suitable text editor of your choice, open the
+.celestia-app file and paste the keyring of the prefunded account.
 
-You can also run the `txsim` Docker image using Docker Compose. Here's an example `docker-compose.yml` file:
+4. We recommend that you set the necessary file permission for the
+.celestia-app file. A simple read access is all that is required for the
+docker container to access the content of the file.
 
-```yaml
-version: '3'
-services:
-  txsim:
-    image: ghcr.io/celestiaorg/txsim
-    command: >
-      -k 0
-      -g consensus-validator-robusta-rc6.celestia-robusta.com:9090
-      -t 10s
-      -b 10
-      -d 100
-      -e 10
-    volumes:
-      - /Users/txsimp/.celestia-app:/home/celestia
-```
+5. You can run the txsim Docker image using the docker run command below.
 
-In this file, the `volumes` key is used to mount the `/Users/txsimp/.celestia-app` directory from the host to the `/home/celestia` directory in the Docker container.
+   ```bash
+   docker run -it \
+   -v $HOME/.celestia-app:/home/celestia ghcr.io/celestiaorg/txsim \
+   -k 0 \
+   -r http://consensus-validator-robusta-rc6.celestia-robusta.com:26657, \
+   http://consensus-full-robusta-rc6.celestia-robusta.com:26657 \
```

Comment:
We can get rid of these two lines, as the tx sim no longer requires an rpc endpoint and the -r flag has been removed